src_cpp/opennnBridge/definitionsNN.h

```diff
@@ -0,0 +1,24 @@
+#pragma once 
+
+enum ModuleType {APPROXIMATION = 1, CLASSIFICATION = 2, FORECASTING = 3 , ENCODER_DECODER = 4, CUSTOMNN = 5};
+
+enum ScalingMethods {NoScaling = 1 , MinimumMaximum = 2 , MeanStandardDeviation = 3 , StandardDeviation = 4 , Logarithm = 5};
```

Comment:
Should start with E prefix (at least) or E_SCALING_METHOD_<method>

Reply:
done